products/apache-http-server.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Apache HTTP Server
+permalink: /apache
+alternate_urls:
+  - /httpd
+iconSlug: apache
+layout: post
+link: https://httpd.apache.org/
+category: server-app
+activeSupportColumn: true
+command: httpd -v
+releaseColumn: false
+releaseDateColumn: true
+sortReleasesBy: "releaseCycle"
+releases:
+  - releaseCycle: "2.4"
+    release: 2012-02-21
+    eol: false
+    latest: 2.4.52
```

Comment:
latest tag is string so could you please change 2.4.52 to "2.4.52"

Reply:
changed it.